aips/aip-lite-account.md

```diff
@@ -0,0 +1,143 @@
+
+## Specification and Implementation Details
+
+This AIP primarily introduces the following new structs:
+
+In `lite_account.move`:
+```
+    #[resource_group(scope = address)]
+    /// A shared resource group for storing new account resources together in storage.
+    struct LiteAccountGroup {}
+
+    #[resource_group_member(group = aptos_framework::lite_account::LiteAccountGroup)]
+    /// Resource representing an account object.
+    struct Account has key {
+        sequence_number: u64,
+    }
+
+    #[resource_group_member(group = aptos_framework::lite_account::LiteAccountGroup)]
+    /// The native authenticator where the key is used for authenticator verification in native code.
+    struct NativeAuthenticator has key, copy, drop {
+        auth_key: Option<vector<u8>>,
+    }
+
+    #[resource_group_member(group = aptos_framework::lite_account::LiteAccountGroup)]
+    /// The dispatchable authenticator that defines how to authenticates this account in the specified module.
+    /// An integral part of Account Abstraction.
+    struct DispatchableAuthenticator has key, copy, drop {
+        auth_function: FunctionInfo
+    }
+
+    #[resource_group_member(group = aptos_framework::lite_account::LiteAccountGroup)]
+    /// Legacy field from deprecated Account module.
+    struct LegacyGUIDCreactionNumber has key {
+        creation_number: u64,
+    }
+
+    #[resource_group_member(group = aptos_framework::lite_account::LiteAccountGroup)]
+    /// Legacy field from deprecated Account module.
+    struct LegacyRotationCapabilityOffer has key, drop { for: address }
+
+    #[resource_group_member(group = aptos_framework::lite_account::LiteAccountGroup)]
+    /// Legacy field from deprecated Account module.
+    struct LegacySignerCapabilityOffer has key, drop { for: address }
+```
+
+And other functions to support all the operations on those resources, please refer to the reference implementation for more details.
+
+
+## Reference Implementation
+
+https://github.com/aptos-labs/aptos-core/pull/10780
+
+## Testing (Optional)
+
+All functionality verified with unit tests across aptos-framework, e2e-move-tests and api.
+
+## Risks and Drawbacks
+
+Since this AIP proposes a compatible solution, if the ecosystem still rely on account.move heavily and do not adopt the features of the lite account, the benefits would be greatly deprecated. 
+
+The migration plan would be two steps:
+1. Stop creating v1 account and only create v2 account on chain.
+2. Opt-in migration to v2 account.
+3. Open to discussion whether global migration is necessary.
+
+## Security Considerations
+
+If not implemented correctly, an account may have both `account::Account` and `lite_account::Account`. Due to circular dependency, we may want to have a native function to verify there is no `account::Account` when creating new lite account.
```

Comment:
We should be able to safely domain-separate between the two, no?

Reply:
finally I decide to put them in one module to avoid those trouble.